### DISCUSSION TITLE
How to toggle "active" state on Navbar dropdown?

### DISCUSSION THREAD

**Discussion by NikarashiHatsu** [saadeghi/daisyui#2617] [Q&A] [answered]
Hi, is there any way to toggle an active state on the dropdown using the `<summary>` tag?

<img width="683" alt="Screenshot 2023-12-05 at 09 54 24" src="https://github.com/saadeghi/daisyui/assets/43662951/81b3d1ee-883a-4270-8c3e-7b9d8b9bfc71">

I tried to apply the `active` class to the `<li>`, the `<summary>`, and the `<details>` tag, and it didn't work. Any workarounds? At least I need an indicator that the menu is toggled, such as using black background color or something like that.

```blade
<li>
  <details class="active">
    <summary>
      <x-phosphor-gear class="w-5 h-5" />
      <span>
        {{ __('layouts/menu.Configurations') }}
      </span>
    </summary>
    <ul class="p-2 !mt-0 w-48">
      <li>
        <a>
          {{ __('layouts/menu.Users') }}
        </a>
      </li>
      ...
    </ul>
  </details>
</li>```

**Comment by saadeghi** [comment, marked as answer]
`<details>` gets an `open` attribute when it's open:  
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#attributes

Let me know if you have a question.

**Reply by NikarashiHatsu** [reply]
No other option rather than using `open` attribute?
I want it to behave like an active menu such this:

```html
<ul class="menu bg-base-200 w-56">
  <li><a>Item 1</a></li>
  <li><a class="active">Item 2</a></li>
  <li><a>Item 3</a></li>
</ul>
```

<img width="202" alt="Screenshot 2023-12-05 at 19 43 06" src="https://github.com/saadeghi/daisyui/assets/43662951/930b5217-bd2c-4485-8fc6-cd376d7963c0">
